### PR TITLE
1843 mdd mdr

### PR DESCRIPTION
**Pull request by Kicksta**
Issue 1843. Add MDD/MDR to product information and add as filter. Change order products

### CODE REVIEW

**Comment by Kicksta** [comment]
@MikeOverkamp-diag, @jmsmkn. I notice two checks failed. It says: "Added line #L%%% was not covered by tests"
Do you have hints on how I can fix this? I was hoping to have this live by tomorrow as the new rules (MDR) comes into effect then and I will send out a relating newsletter on Thursday. Thanks for your help!

**Comment by jmsmkn** [comment]
You don't need to worry about those two, they are not required.

@MikeOverkamp-diag could you review this? Thanks!

File: app/grandchallenge/products/models.py [hunk 1]
```diff
@@ -113,6 +113,7 @@ class FDAStatus(models.TextChoices):
     ce_status = models.CharField(
         choices=CEStatus.choices, max_length=3, default=CEStatus.NO
     )
+    ce_under = models.CharField(max_length=10, default="")
```
Comment (MikeOverkamp-diag) [inline app/grandchallenge/products/models.py:116]: You could also use `blank=True` instead of `default=""`. That is a bit more common.

**Review by MikeOverkamp-diag — COMMENTED** [review]
One minor comment.

**Review by MikeOverkamp-diag — APPROVED** [review]

**Pull request merged**